Cython/Tests/TestCodeWriter.py

```diff
@@ -107,6 +107,18 @@ def test_cascaded_assignment(self):
     def test_attribute(self):
         self.t(u"a.x")
 
+    def test_return_none(self):
+        self.t(u"""
+                    def f(x, y, z):
+                        return
+                    cdef f(x, y, z):
+                        return
+                    def f(x, y, z):
```

Comment:
I think it should work fine, but can we have a `return None` test in here too?

Reply:
Now such cases has been covered as well.